src/SFML/Window/OSX/SFOpenGLView.h

```diff
@@ -71,8 +71,7 @@ class WindowImplCocoa;
 /// the cursor (that was disconnected from the system).
 ///
 ////////////////////////////////////////////////////////////
-@interface SFOpenGLView : NSOpenGLView
-{
+@interface SFOpenGLView : NSOpenGLView {
```

Comment:
I think you somehow accidentally used clang-format-15 to format this which sadly doesn't work. If you used the `format` target to do the formatting, it should have detected that you were using v15 and prevented it from running.

Reply:
As mentioned in private it was that I forced CLion to do formatting.